whylog/config/parsers.py

```diff
@@ -24,14 +23,24 @@ def __init__(self, name, regex, primary_key_groups, log_type, convertions):
     def get_clue(self, line):
         pass
 
-    def serialize_parser(self):
-        return {
-            "name": self.name,
-            "regex": self.regex_str,
-            "primary_key_groups": self.primary_key_groups,
-            "log_type": self.log_type,
-            "convertions": self.convertions
-        }
+    def to_data_access_object_form(self):
+        return RegexParserDAO(
+            self.name, self.regex_str, self.primary_key_groups, self.log_type, self.convertions
+        )
+
+
+class RegexParserDAO(object):
+    def __init__(self, name, regex, primary_key_groups, log_type, convertions):
+        self.name = name
```

Comment:
will Ewa pass name to a regex? I don't think she will. Where would she get it from?